### PR TITLE
[docs] Use react-standalone

### PR DESCRIPTION
**Pull request by zpao**
I missed that #5497 got closed and babel-standalone supports what I wanted, so we can do this now.

### CODE REVIEW

**Comment by Daniel15** [comment]
Thanks! I've been meaning to update React to use babel-standalone in the examples. Thanks for working on it 😄

File: docs/docs/02-displaying-data.it-IT.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ Diamo un'occhiata ad un esempio davvero semplice. Creiamo un file dal nome `hell
     <title>Hello React</title>
     <script src="https://unpkg.com/react@{{site.react_version}}/dist/react.js"></script>
     <script src="https://unpkg.com/react-dom@{{site.react_version}}/dist/react-dom.js"></script>
-    <script src="https://unpkg.com/babel-core@5.8.38/browser.min.js"></script>
+    <script src="https://unpkg.com/babel-standalone@6.15.0/babel.min.js"></script>
```
Comment (Daniel15) [inline docs/docs/02-displaying-data.it-IT.md]: Babel-standalone is on CDNJS and I've seen better perf with CDNJS compared to unpkg (also it has more users so it's more likely to hit the cache). Maybe use CDNJS instead?

File: docs/docs/09.2-package-management.md [hunk 2]
```diff
@@ -99,7 +99,7 @@ bower install --save react
     <title>Hello React!</title>
     <script src="bower_components/react/react.js"></script>
     <script src="bower_components/react/react-dom.js"></script>
-    <script src="https://unpkg.com/babel-core@5.8.38/browser.min.js"></script>
+    <script src="https://unpkg.com/babel-standalone@6.15.0/babel.min.js"></script>
```
Comment (Daniel15) [inline docs/docs/09.2-package-management.md]: In this one, you might want to use babel-standalone from Bower for consistency. Updating the install command to `bower install --save react babel-standalone` should work.

**Comment by Daniel15** [comment]
Instead of using `babel-standalone@6.15.0`, you could use `babel-standalone@6` to use the latest 6.x.x version if you like.

**Comment by aweary** [comment]
Looks like this super out of date, and was addressed in https://github.com/facebook/react/pull/7868

**Pull request closed without merging**